extras/capitanodoc/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { OptionDefinition } from 'capitano';
+import * as _ from 'lodash';
+import * as ent from 'ent';
+
+export function getOptionPrefix(signature: string) {
+	if (signature.length > 1) {
+		return '--';
+	} else {
+		return '-';
+	}
+};
+
+export function getOptionSignature(signature: string) {
+	return `${getOptionPrefix(signature)}${signature}`;
+}
+
+export function parseSignature(option: OptionDefinition) {
+	let result = getOptionSignature(option.signature);
+
+	if (!_.isEmpty(option.alias)) {
+		if (_.isString(option.alias)) {
+			result += `, ${getOptionSignature(option.alias)}`;
+		} else {
+			for (let alias of option.alias!) {
```

Comment:
I'm not sure what this means. Which `if` - the `isString`, or `isEmpty`? I can't see how we can easily change either one to fix this... I could change the `else` to an explicit `else if` and check there, but that changes what this code actually does slightly.

Really, TS should know this can't be undefined because of the `isEmpty`, but the lodash typings don't include a type guard for that (and I think it's impossible to write a 100% correct one, sadly).

Reply:
Opps, sorry for being confusing... I was meaning something like:
```ts
if (_.isArray(option.alias)) {
	for (let alias of option.alias) { /*...*/ }
} else {
	result += `, ${getOptionSignature(option.alias)}`;
}
```
but it's an NIT anyway.

Reply:
Ah, got it. Fixed!